client/src/backoffice/EventsPage.tsx

```diff
@@ -169,44 +174,101 @@ const EventField: React.FC<EventFieldProps> = ({ title, name, disabled }) => {
 
 const EventList: React.FC = () => {
   const [events, fetchingEvents, fetchEventsError] = useAsync(getEvents);
+  const [criteria, setCriteria] = useState<string>('');
 
   return (
-    <div className={"bk-container"}>
+    <div className={'bk-container'}>
       <h2>Events</h2>
       <Link to="/admin/events/new">
         <button className="bk-btn" style={{ margin: '30px 0px' }}>
           Create New
         </button>
       </Link>
-      {fetchingEvents ? (
-        <div>Fetching Events...</div>
-      ) : fetchEventsError || events == null ? (
-        <div>There was a problem fetching events</div>
-      ) : (
-        <div>
-          {events.map(e => (
-            <EventRow key={e.id} event={e} />
-          ))}
-        </div>
-      )}
+      <input
+        type="text"
+        placeholder="Search by name"
+        onChange={e => setCriteria(e.target.value.toLowerCase())}
+      />
+      {fetchingEvents && <Loading />}
+
+      {(fetchEventsError || events === null) && <div>There was a problem fetching events</div>}
+
+      {events !== null && <EventTable criteria={criteria} events={events} />}
     </div>
   );
 };
 
-const EventRow: React.FC<{ event: PoapEvent }> = ({ event }) => {
+type PaginateAction = {
+  selected: number;
+};
+
+type EventTableProps = {
+  events: PoapEvent[];
+  criteria: string;
+};
+
+const EventTable: React.FC<EventTableProps> = ({ events, criteria }) => {
+  const total = events.length;
+  const pageRange = 10;
```

Comment:
There's a global const in line [29](https://github.com/Xivis/poap/pull/43/files#diff-5138b897ff09d68570ac90950c2c515fR29) should we use that since it has the same value ?